circleci.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+set -uexo pipefail
+
+HOST_DMD_VER=2.068.2 # same as in dmd/src/posix.mak
+CURL_USER_AGENT="CirleCI $(curl --version | head -n 1)"
+N=2
+
+case $CIRCLE_NODE_INDEX in
+    0) MODEL=64 ;;
+    1) MODEL=32 ;;
+esac
```

Comment:
I asked @wilzbach to stick with one, because the Terms Of Service only offer 4 instances for open source projects. It was unclear what CircleCI's definition of open source was, so while we await clarification I'd like to stay with one. (If their definition is "public repository", then we're golden.)